src/test/suite/didactUriCompletionItemProvider.test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import { expect } from 'chai';
+import { DidactUriCompletionItemProvider } from "../../didactUriCompletionItemProvider";
+
+suite("Didact URI completion", function () {
+
+	let emptyContent = '';
+	let notEmptyContent = 'this is text with a didact://? link stuffed inside';
+
+	test("no result for uri completion when no didact link in line", () => {
+		expect(new DidactUriCompletionItemProvider().provideCompletionItemsForDidactProtocol(emptyContent)).to.be.empty;
+	});
+
+	test("should get result for uri completion when didact link in line", () => {
+		expect(new DidactUriCompletionItemProvider().provideCompletionItemsForDidactProtocol(notEmptyContent)).not.empty;
+	});
+
+	test("should get result for outside uri completion when no didact link in line", () => {
+		expect(new DidactUriCompletionItemProvider().provideCompletionItemsOutsideDidactURI(emptyContent)).not.empty;
```

Comment:
maybe we can improve a bit the test and checs for the number of expected completion?